### ISSUE TITLE
Accessibility: MAS42A:  In the high contrast black/white theme, there is no proper border for Search Docs box.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#9]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.
**_2. Change system setting to High contrast Black_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast Black" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
1. Open https://code.visualstudio.com/ in Edge.
2. Navigate to the "Search Docs" box beside the download link in header of the page .
3. Observe the "Search Docs" box border, High contrast black/white theme.

Note: Same issue is observed for the below link
https://code.visualstudio.com/docs
http://code.visualstudio.com/Download
http://code.visualstudio.com/insiders
https://code.visualstudio.com/blogs
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
The "Search Docs " box is not having the proper border in high contrast black and white  theme

**Expected Result:**
There should be proper border for the "Search Docs" box in High contrast black and white theme.

**User Impact:**
Low vision user might not be knowing separation of button if borders are missing.
[Search box boarder.zip](https://github.com/wiproaccess/vscode-website/files/686926/Search.box.boarder.zip)

**Comment by waderyan** [comment]
Fixed
